### PR TITLE
🌐 Add French translation for `docs/fr/docs/tutorial/testing.md`

### PR DESCRIPTION
**Pull request by NastasiaSaby**
Hi!

Here is the PR to translate the tutorial/testing into French.
See the French translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews 👌

### CODE REVIEW

File: docs/fr/docs/tutorial/testing.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    Remarquez que les fonctions de tests commencent normalement par `def` et non `async def`.
+
+    Et les appels au client sont des appels normaux n'utilisant pas `await`.
+
+    Cela permet d'utiliser `pytest` directement sans problème.
+
+!!! note "Détails techniques"
+    Vous pouvez aussi utiliser `from starlette.testclient import TestClient`.
+
+    **FastAPI** fournit `starlette.testclient` derrière `fastapi.testclient`. `fastapi.testclient` a été créé juste pour votre comodité à vous développeur, mais il vient directement de Starlette.
```
Comment (Smlep) [inline docs/fr/docs/tutorial/testing.md:35]:
```suggestion
    **FastAPI** fournit `starlette.testclient` derrière `fastapi.testclient`. `fastapi.testclient` a été créé juste pour votre commodité à vous développeur, mais il vient directement de Starlette.
```

File: docs/fr/docs/tutorial/testing.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+Disons que vous avez un fichier `main.py` avec votre app **FastAPI** :
+
+```Python
+{!../../../docs_src/app_testing/main.py!}
+```
+
+### Le fichier de test
+
+Ensuite, vous pourriez avoir un fichier `test_main.py` avec vos tests et importer votre `app` depuis le module `main` (`main.py`):
```
Comment (Smlep) [inline docs/fr/docs/tutorial/testing.md:56]:
```suggestion
Ensuite, vous pourriez avoir un fichier `test_main.py` avec vos tests et importer votre `app` depuis le module `main` (`main.py`) :
```

File: docs/fr/docs/tutorial/testing.md (hunk 1 above)
Comment (NastasiaSaby) [reply docs/fr/docs/tutorial/testing.md]: Thank you. Done :).

File: docs/fr/docs/tutorial/testing.md (hunk 2 above)
Comment (NastasiaSaby) [reply docs/fr/docs/tutorial/testing.md]: Thank you. Done :).

**Review by Smlep — APPROVED** [review]

**Review by AlexandreBiguet — COMMENTED** [review]
Hi, thanks for the translation!

I added some comments / suggestions, let me know what you think :)

**Comment by tiangolo** [comment]
📝 Docs preview for commit f878130c5cf38bc4119a96f88140067d49115cff at: https://6499e2bff8ac9505a95fb768--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**